user-group/survey01-email.html

```diff
@@ -0,0 +1,114 @@
+<body style="margin:0;padding:0">
+	<div style="Margin:0;padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;min-width:100%;background-color:#EAECEC;">
+		<table style="border-spacing:0;width:100%;table-layout:fixed;">
+			<tbody>
+				<tr>
+					<td align="center" style="padding-top:30px;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;">
+
+						<center>
+
+							<table style="border-spacing:0;Margin:0 auto;" width="600">
+								<tbody>
+									<tr>
+										<td bgcolor="#111419" style="padding-top:60px;padding-bottom:30px;padding-right:0;padding-left:0;vertical-align:top;Margin:0 auto;text-align:center;" width="600">
+											<h1>
+												<img alt="Ally logo" height="102px" src="/surveys/img/ally-logo.png" style="Margin-bottom:10px;" title="Ally" />
+											</h1>
+										</td>
+									</tr>
+									<tr>
+										<td bgcolor="#111419" style="padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;vertical-align:top;Margin:0 auto;" width="600">
+											<table style="border-spacing:0;" width="100%">
+												<tbody>
+													<tr>
+														<td style="text-align:center;padding-top:0;padding-bottom:0;padding-right:60px;padding-left:60px;vertical-align:top">
+															<h3 style="font-family:sans-serif;font-weight:lighter;text-align:center;font-size:32px;line-height:28px;Margin-top:0;Margin-bottom:10px;color:#00c0cb;"><span style="color:#00C0CB">Take the User Group Survey</span></h3>
+															<h4 style="font-family:sans-serif;font-weight:lighter;text-align:center;font-size:22px;line-height:28px;Margin-top:0;Margin-bottom:30px;color:#ffffff;"><span style="color:white">Help us understand how your institution supports accessibility with 12 questions.</span></h4>
+														</td>
+													</tr>
+													<tr>
+														<td style="text-align:center;padding-top:0;padding-bottom:0;padding-right:60px;padding-left:60px;vertical-align:top;height:80px">
+															<a href="[SurveyLink]" target="_blank" style="padding-top:10px;padding-bottom:10px;padding-right:20px;padding-left:20px;background-color:#007900;font-family:sans-serif;font-weight:lighter;font-size:22px;line-height:22px;text-align:left;text-decoration: none;color:#ffffff;">Take the survey now</a>
+														</td>
+													</tr>
+													<tr>
+														<td style="text-align:center;padding-top:0;padding-bottom:0;padding-right:60px;padding-left:60px;height:225px;">
+															<img alt="Picture of survey interface" width="382px" height="225px" src="/surveys/img/survey-snap.png" title="Survey" />
```

Comment:
Also, this should be `alt=""` as it's decorative. The title can be removed as well